src/spawn.c

```diff
@@ -67,6 +67,7 @@
 #else
 # include "support.h"
 #endif
+#include "utils.h"
```

Comment:
It seems only `utils_get_utf8_from_locale` is used from utils.c in Windows-only code paths and this function is just a wrapper for `g_strdup` on Windows. So it might be OK to just use `g_strdup`.